e2e/cypress/integration/api/iam/iam_integration.spec.ts

```diff
@@ -148,6 +148,23 @@ describeIfIAMV2p1('assigning projects', () => {
       });
     });
 
+    it(`when creating ${iamResources} without projects is allowed`,  () => {
```

Comment:
in keeping with the other test names, I'd include the expected behavior:
`when creating ${iamResources} without projects is allowed, successfully creates unassigned ${iamResource}`